packages/react/src/auto/hooks/useTableBulkActions.tsx

```diff
@@ -15,14 +15,15 @@ export type ModelActionDetails =
   | {
       isGadgetAction: false;
       apiIdentifier: string;
-      render?: (recordIds: string[], records: any[]) => React.ReactNode;
+      onPromptComponent?: onPromptComponent;
```

Comment:
This feels like a mouthful. How about `promptContent`?

Reply:
Updated to be `promptContent` throughout